redis.conf

```diff
@@ -883,6 +883,40 @@ replica-priority 100
 #
 # Basically ACL rules are processed left-to-right.
 #
+# The following is a list of command categories and their meanings:
+# * keyspace - Writing or reading from keyspace, keys or their metadata but not
+#     a certain data type. Includes DEL, RESTORE, DUMP, RENAME, EXISTS, DBSIZE,
```

Comment:
Since a keyspace isn't a super well defined concept in Redis:
```suggestion
# * keyspace - Writing or reading from keys, databases, or their metadata 
#     in a type agnostic way. Includes DEL, RESTORE, DUMP, RENAME, EXISTS, DBSIZE,
```
```